[2025-01-09T14:40:56.847+0000] {logging_mixin.py:149} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-01-09T14:40:56.851+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: config_loader_dag.load_config_task scheduled__2025-01-08T00:00:00+00:00 [queued]>
[2025-01-09T14:40:56.854+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: config_loader_dag.load_config_task scheduled__2025-01-08T00:00:00+00:00 [queued]>
[2025-01-09T14:40:56.854+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2025-01-09T14:40:56.859+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): load_config_task> on 2025-01-08 00:00:00+00:00
[2025-01-09T14:40:56.861+0000] {standard_task_runner.py:57} INFO - Started process 196 to run task
[2025-01-09T14:40:56.862+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'config_loader_dag', 'load_config_task', 'scheduled__2025-01-08T00:00:00+00:00', '--job-id', '6', '--raw', '--subdir', 'DAGS_FOLDER/config_loader_dag.py', '--cfg-path', '/tmp/tmpr14fygyr']
[2025-01-09T14:40:56.863+0000] {standard_task_runner.py:85} INFO - Job 6: Subtask load_config_task
[2025-01-09T14:40:56.870+0000] {logging_mixin.py:149} WARNING - /home/***/.local/lib/python3.7/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-01-09T14:40:56.888+0000] {task_command.py:410} INFO - Running <TaskInstance: config_loader_dag.load_config_task scheduled__2025-01-08T00:00:00+00:00 [running]> on host cdb64526c5e5
[2025-01-09T14:40:56.894+0000] {logging_mixin.py:149} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-01-09T14:40:56.923+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='config_loader_dag' AIRFLOW_CTX_TASK_ID='load_config_task' AIRFLOW_CTX_EXECUTION_DATE='2025-01-08T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-01-08T00:00:00+00:00'
[2025-01-09T14:40:56.923+0000] {logging_mixin.py:149} INFO - Parameter 1: Hello, Airflow with Config!
[2025-01-09T14:40:56.924+0000] {logging_mixin.py:149} INFO - Parameter 2: 42
[2025-01-09T14:40:56.924+0000] {python.py:183} INFO - Done. Returned value was: None
[2025-01-09T14:40:56.928+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=config_loader_dag, task_id=load_config_task, execution_date=20250108T000000, start_date=20250109T144056, end_date=20250109T144056
[2025-01-09T14:40:56.964+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2025-01-09T14:40:56.972+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
